The stock gtk-doc macros aren't used, so gtkdocize is not an option

--- p11-kit-0.18.7-orig/build/gtk-doc.make	2013-10-14 07:15:57.000000000 -0500
+++ p11-kit-0.18.7/build/gtk-doc.make	2014-01-24 01:10:15.790983600 -0600
@@ -257,3 +257,5 @@
 	@$(GTKDOC_REBASE) --online --relative --html-dir=$(distdir)/html
 
 .PHONY : dist-hook-local docs
+
+.NOTPARALLEL:
